starter-code/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import './App.css';
+import Item from './Item';
+
+class App extends Component {
+	render() {
+		let Item1 = {
+			imgURL: '/images/icon1.png',
+			title: 'Declarative',
+			text: 'Rect makes it painless to create interactive UIs'
+		};
+
+		let Item2 = {
+			imgURL: '/images/icon2.png',
+			title: 'Components',
+			text: 'Build encapsulated components that manage their state;'
+		};
+
+		let Item3 = {
+			imgURL: '/images/icon3.png',
+			title: 'Single-Way',
+			text: "A set of immutable values are passed to the component's"
+		};
+
+		let Item4 = {
+			imgURL: '/images/icon4.png',
+			title: 'JSX',
+			text: 'Statically typed designed to run on modern browsers'
+		};
+
+		return (
+			<div className="App">
+				<img class="logo1" src="/images/ironhack-logo.svg" />
+				<img class="logo2" src="/images/menu-top.svg" />
+				<img class="logo3" src="/images/react-logo.svg" />
+				<div className="header">
+					<h1>
+						Say Hello to <br /> ReactJS
+					</h1>
+					<p>
+						You will learn a Frontend <br /> framework from scratch to <br /> become an Ninka Developer
+					</p>
+					<button type="button" class="btn btn-light">
+						Awesome!
+					</button>
+				</div>
```

Comment:
I think the header could be a component as well.  The nice thing about making components in separate files is that you can avoid hard to read or complex length of code.  If your nav, header and items are all in separate files your `App.js` can focus on passing data to its various components and it'll be a lot easer to read and maintain